docs/platform/concepts/backup-to-another-region.rst

```diff
@@ -0,0 +1,79 @@
+About backup to another region for Aiven services
+=================================================
+
+.. important::
+
+    Backup to another region (BTAR) is available on ``:doc:Pro Platform </docs/platform/concepts/pro-platform>``.
+
+Discover the backup to another region (BTAR) feature, check why use it, and learn how it works.
+
+About BTAR
+----------
+
+The backup to another region (BTAR) feature allows existing backup files to be copied from the service's primary backup region to one or more regions different from the primary one. As a disaster recovery feature, BTAR is particularly useful when the service-hosting region is down since it allows forking the service from an additional copy of the backup residing outside the service-hosting region.
+
+BTAR is currently supported for the following services:
+
+* Aiven for PostgreSQL®
+* Aiven for MySQL®
+
+Why use BTAR
+------------
+
+BTAR provides an additional level of security for your data. It is a cost-efficient low-complexity disaster recovery feature that allows you to spread your data across different regions and recovery it in case of data loss or corruption in the primary region. During an outage or a downtime on the primary region, your data is protected and safe in a secondary backup location.
```

Comment:
```suggestion
BTAR provides an additional level of resilience for your data. It is a cost-efficient, low-complexity disaster recovery feature that allows you to replicate your data in a different region and recover there in case of outage, downtime, data loss or corruption in the primary region.
```